### ISSUE TITLE
AttributeError: 'DAL' object has no attribute 'adapter_args'

### ISSUE THREAD

**Issue by diegoteixeir4** [gi0baro/weppy#121]
I'm getting this error when i try to perform migrations against a MySQL database, but it works with SQLite.

```
/weppy-app/env/lib/python2.7/site-packages/pydal/adapters/base.py:1379: Warning: Changing sql mode 'NO_AUTO_CREATE_USER' is deprecated. It will be removed in a future release.
  ret = self.get_cursor().execute(command, *a[1:], **b)
> Performing upgrades against mysql://root@localhost/weppy_app
> Performing upgrade: <base> -> 24f72b05fd3 (head), Create tables
> [ERROR] failed upgrading to <base> -> 24f72b05fd3 (head), Create tables
Traceback (most recent call last):
  File "/weppy-app/env/bin/weppy", line 11, in <module>
    sys.exit(main())
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/cli.py", line 358, in main
    cli.main(args=args, prog_name=name)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/cli.py", line 209, in main
    return click.Group.main(self, *args, **kwargs)
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/weppy-app/env/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/weppy-app/env/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/cli.py", line 326, in migrations_up
    up(app, dals, revision)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/commands.py", line 230, in up
    Command(app, dals).up(revision)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/commands.py", line 172, in up
    migration.up()
  File "/weppy-app/migrations/24f72b05fd3_create_tables.py", line 23, in up
    migrations.Column('password', 'string', length=255)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/helpers.py", line 37, in __call__
    return op.run()
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/operations.py", line 125, in run
    self.engine.create_table(self.table_name, self.columns, **self.kw)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/engine.py", line 65, in create_table
    sql = self._new_table_sql(name, columns, **params)
  File "/weppy-app/env/lib/python2.7/site-packages/weppy/dal/migrations/engine.py", line 218, in _new_table_sql
    engine = self.db.adapter_args.get('engine', 'InnoDB')
  File "/weppy-app/env/lib/python2.7/site-packages/pydal/base.py", line 921, in __getattr__
    return BasicStorage.__getattribute__(self, key)
AttributeError: 'DAL' object has no attribute 'adapter_args'
```

How can i solve this?

**Comment by gi0baro** [comment]
@diegoteixeir4 which version of weppy are you using?
Will inspect this asap.

**Comment by gi0baro** [comment]
@diegoteixeir4 Yes this is confirmed on the latest release, this line https://github.com/gi0baro/weppy/blob/release/weppy/dal/migrations/engine.py#L218 is wrong.

Will release an hotfix by tomorrow.

**Comment by gi0baro** [comment]
@diegoteixeir4 This is fixed on v0.6.5, please update your copy. The master was already fixed.
